src/Elmish.WPF/ViewModel.fs

```diff
@@ -357,25 +348,22 @@ and [<AllowNullLiteral>] internal ViewModel<'model, 'msg>
     | Cmd _
     | CmdParam _ ->
         false
-    | SubModel ((vm: ViewModel<obj, obj> voption ref), (getModel: 'model -> obj voption), getBindings, toMsg, sticky) ->
+    | SubModel (vm, getModel, getBindings, toMsg, sticky) ->
       match !vm, getModel newModel with
       | ValueNone, ValueNone -> false
       | ValueSome _, ValueNone ->
-          if sticky then false
```

Comment:
Please keep this as it is

Reply:
Sure, reverted.

I "can't" even argue for consistency since I put `then` on its own line just as `else` is on its own line but then didn't also put `false` on its on line like the body of `else` is :P

And I definitely don't want `then \newline false`.  That is way to verbose reguarless of parallel structure.